test/spec/modules/symitriDapRtdProvider_spec.js

```diff
@@ -260,6 +268,34 @@ describe('symitriDapRtdProvider', function() {
     });
   });
 
+  describe('dapX2Tokenize', function () {
+    it('dapX2Tokenize error callback', function () {
+      let configAsync = JSON.parse(JSON.stringify(sampleX2Config));
+      let submoduleCallback = dapUtils.dapTokenize(configAsync, sampleIdentity, onDone,
+        function(token, status, xhr, onDone) {
+        },
+        function(xhr, status, error, onDone) {
+        }
+      );
+      let request = server.requests[0];
+      request.respond(400, responseHeader, JSON.stringify('error'));
+      expect(submoduleCallback).to.equal(undefined);
```

Comment:
You're just testing for undefined callback; it seems your pr shows changing the endpoint, you should test if changing the config successfully changes the endpoint destination